### ISSUE TITLE
[Feature request] Bulk delete cards

### ISSUE THREAD

**Issue by RoamanEmpire** [Pseudonium/Obsidian_to_Anki#234]
An indicator at the top of a note like "DELETE ALL" to delete all notes in a document would be very useful. I know it is possible to delete the notes in Anki and the remove them from their respective notes. However, this is complicated if someone uses a single deck for cards imported from multiple notes; then finding each card in the Anki browser becomes very time consuming.

**Comment by ravipawar89** [comment]
yes this is much needed. it would be better to delete all notes from obsidian in one go
